Pipfile

```diff
@@ -11,10 +11,3 @@ name = "pypi"
 
 [packages]
 
-scrapy = "*"
```

Comment:
Looks like you removed some required dependencies from the Pipfile. Can you remove this change and the `Pipfile.lock` changes from this PR?